extensions/applicationinsights-angular-js/test-app/src/app/app.component.spec.ts

```diff
@@ -92,23 +81,30 @@ describe('Router: App', () => {
         },
         }
     };
+    
     core.initialize(config, [angularPlugin, analyticsExtension, channel]);
-    // spy on track
-    const spy = spyOn(angularPlugin, 'trackPageView');
 
-    // Emulate navigation to different URL-addressed pages
-    // navigate to /home
-    router.navigate([''])
-    .then(() => {
-      tick(500);
-      expect(angularPlugin.trackPageView).toHaveBeenCalledTimes(1);
-    });
+    // trackPageView is called on plugin intialize
+    // default route is '/'
+    expect(angularPluginTrackPageViewSpy).toHaveBeenCalled();
+    expect(angularPluginTrackPageViewSpy).toHaveBeenCalledWith({ uri: '/' } as IPageViewTelemetry);
 
+    // Emulate navigation to different URL-addressed pages
     // navigate to /search
     router.navigate(['search'])
     .then(() => {
       tick(500);
-      expect(angularPlugin.trackPageView).toHaveBeenCalledTimes(1);
+      expect(angularPluginTrackPageViewSpy).toHaveBeenCalledTimes(2);
+      expect(angularPluginTrackPageViewSpy).toHaveBeenCalledWith({ uri: '/search' } as IPageViewTelemetry);
+    })
+    // navigate to /home
+    .then(() => {
+      router.navigate(['home'])
```

Comment:
navigate to /home needs to happen after the first navigate done, otherwise uri will always be the second one.